src/app/fyle/dashboard/dashboard.service.ts

```
@@ -46,6 +51,33 @@ export class DashboardService {
       );
   }
 
+  getUnapprovedTeamReportsStats(showTeamReportStats: Boolean): Observable<PlatformReportsStatsResponse> {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)